### DISCUSSION TITLE
Disable automatic quoting of text entered into Data editor cell in generated insert/update query

### DISCUSSION THREAD

**Discussion by xitiroc** [dbeaver/dbeaver#17977] [Q&A]
I need the value entered into a cell in the Data editor for a column to be sent as-is rather than being surrounded by single quotes in the insert or update query. Is there a way to accomplish this?

For example, if I enter `uuid()` into a cell, I need the resulting query to be `UPDATE MyTable SET user_uuid=uuid() WHERE user_id=1;` instead of  `UPDATE MyTable SET user_uuid='uuid()' WHERE user_id=1;`

![image](https://user-images.githubusercontent.com/116259303/196892440-6e59f360-256f-4fa3-a96d-a17bedaffa34.png)

Using: 
App Version - DBeaverCE_22.2.0.0
OS - Windows 11
Driver - org.mariadb.jdbc:mariadb-java-client:RELEASE
DB - MariaDB Server 1:10.9.3

**Comment by LonwoLonwo** [comment]
Hello @xitiroc 

Nope, DBeaver doesn't have such an option. You can copy your script and change it in the SQL Editor.
